docs/docsite/rst/index.rst

```diff
@@ -5,36 +5,32 @@ Ansible AWX helps teams manage complex multi-tier deployments by adding control,
 
 .. toctree::
    :maxdepth: 2
-   :caption: AWX Quickstart
+   :caption: Get started
 
    quickstart/index
 
 .. toctree::
    :maxdepth: 2
-   :caption: User Guide
+   :caption: Community
 
-   userguide/index
+   contributor/index
 
 .. toctree::
    :maxdepth: 2
-   :caption: AWX Administration
+   :caption: Users
```

Comment:
This caption should be Administrators

Reply:
I moved things around a little in the top-level index. We can adjust but each caption matches the corresponding index. In this case line 22 is `userguide/index` and the left-hand nav renders like this:

![image](https://github.com/ansible/awx/assets/32749632/de1d9249-ab07-4aa5-966e-201c54d5134a)